src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --header-size: 60px;
+}
+
 body {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
```

Comment:
You've used 'width: 100%' for '.header'. This is not wrong, but it's unnecessary because block level elements like div (which I assume you're using for '.header') take up the full width available by default.